uiplib/gui/settingsTab.py

```diff
@@ -96,12 +96,13 @@ def create_settings_tab(appObj):
 
     # Number to days to auto-delete
     appObj.auto_del_val = StringVar()
-    appObj.auto_del_val.set(int(int(appObj.settings['days-to-autodel'])))
+    val = int(int(appObj.settings['days-to-autodel'])/(24*60*60))
```

Comment:
Why set it as `int` here? Do it while you save it. Because, there are chances for Exceptions, here.